frontend/src/entities/jobs/algorithms.tsx

```diff
@@ -1250,15 +1608,19 @@ export const ML_ALGORITHMS: Algorithm[] = [
                 key: 'enc0_layers',
                 value: ['auto'],
                 ...linearIntProperties,
-                zValidator: rangeValidator('enc0_layers', 1, 4, false, false, true),
+                zValidator: numberValidator('enc0_layers',
+                    {   
+                        min: 1, 
+                        max: 100,
```

Comment:
```suggestion
                        max: 4,
```
Unless this was previously incorrect.